### PR TITLE
[13.0][ADD] partner_manual_rank

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Partner Manual Rank
----------------------------
This module adds the ability to manually flag partners as customer or supplier from the partner form view.

### CODE REVIEW

File: partner_manual_rank/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    is_customer = fields.Boolean(
+        compute="_compute_is_customer",
+        inverse="_inverse_is_compute",
```
Comment (JordiMForgeFlow) [inline partner_manual_rank/models/res_partner.py]: `inverse="_inverse_is_customer"`

Comment (MiquelRForgeFlow) [reply partner_manual_rank/models/res_partner.py]: my bad 😂

Comment (MiquelRForgeFlow) [reply partner_manual_rank/models/res_partner.py]: done

**Review by JordiMForgeFlow — APPROVED** [review]

**Review by JordiBForgeFlow — APPROVED** [review]

**Pull request merged**